### DISCUSSION TITLE
why the webpack did not generate the css file

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [webpack/webpack#15227] [Q&A] [answered]
I upgrade the webpack to 5.x but I found it did not generate the css output file:

https://stackoverflow.com/questions/70810839/why-the-webpack-5-x-did-not-generate-the-css-file

what should I do to fix the problem?

**Comment by alexander-akait** [comment, marked as answer]
Does answer help you https://stackoverflow.com/questions/70810839/why-the-webpack-5-x-did-not-generate-the-css-file?
